.travis.yml

```diff
@@ -12,6 +12,8 @@ before_install:
 - gem update --system
 - gem --version
 - bundle --version
+env:
+  - SLOW=1
```

Comment:
I like this, and that the slow tests get marked as skipped so we don't forget about them =)

Reply:
Exactly. That was the intent.

If you know of a better way to globally set an env on appveyor, I'd appreciate it.